wai-aria/subtree/tablist.tentative.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/wai-aria/scripts/aria-utils.js"></script>
+
+<div id="tablist" role="tablist" aria-label="tablist">
```

Comment:
mega-nit: might be nice to have the role and label be distinct strings to avoid (my) confusion.